invenio_circulation/utils.py

```diff
@@ -10,22 +10,86 @@
 
 from datetime import datetime, timedelta
 
+from flask import current_app
+
 
 def item_location_retriever(item_pid):
     """."""
     pass
 
 
-def is_checkout_valid(transaction_user_pid,
-                      patron_pid,
-                      item_pid,
-                      transaction_location_pid,
-                      transaction_date,
-                      start_date=None,
-                      end_date=None):
+def is_checkout_valid(
+    transaction_user_pid,
+    patron_pid,
+    transaction_location_pid,
+    transaction_date,
+    item_pid,
+    start_date=None,
+    end_date=None,
+):
     """."""
     if not start_date:
-        start_date = datetime.strptime(transaction_date, '%Y-%m-%d')
+        start_date = transaction_date
+    # 30 days by default
+    if not end_date:
+        end_date = datetime.strptime(start_date, '%Y-%m-%d') + timedelta(
+            days=30
```

Comment:
Is it worth it to make the 30 days by default in a config variable for quick overridability?